spec/core/filtering/simple_rule_spec.rb

```diff
@@ -0,0 +1,199 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+
+# frozen_string_literal: true
+require 'active_support/time'
+require 'core/filtering/simple_rule'
+
+describe Core::Filtering::SimpleRule do
+  subject {
+    Core::Filtering::SimpleRule.new(
+      {
+        Core::Filtering::SimpleRule::ID => id,
+        Core::Filtering::SimpleRule::FIELD => field,
+        Core::Filtering::SimpleRule::VALUE => value,
+        Core::Filtering::SimpleRule::POLICY => policy,
+        Core::Filtering::SimpleRule::RULE => rule
+      }
+    )
+  }
+  let(:id) { 'test' }
+  let(:field) { 'str' }
+  let(:value) { 'foo' }
+  let(:policy) { Core::Filtering::SimpleRule::Policy::INCLUDE }
+  let(:rule) { Core::Filtering::SimpleRule::Rule::EQUALS }
+  let(:document) do
+    {
+      'str' => 'foo',
+      'int' => 123,
+      'time' => time_now,
+      'datetime' => date_now,
+      'bool' => false
+    }
+  end
+  let(:date_now) { DateTime.now.new_offset(0) }
```

Comment:
Curious - why `new_offset(0)`?

Reply:
When I was running the tests, it was using my timezone (`-06:00`). That made it hard for me to write an `ends_with` test that would work on other people's machines. The `new_offset(0)` forces it to be GMT (`-00:00`).

Honestly though, IDK why our spec_helper isn't forcing this already. 🤔